app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife \
```

Comment:
All good , but please try to remove this breaking lines ( \ ) . Maybe try to put duplicated code into a viriable  `Person.people[person["name"]]` ??